example/server.js

```diff
@@ -8,24 +8,18 @@ const webpackServeBundle = require('./webpackServeBundle');
 const app = express();
 const isDevelopment = process.env.NODE_ENV !== 'production';
 
+const middlewarePath = '/falcor-postman';
+const falcorModelPath = '/example.json';
+
+const options = { middlewarePath, falcorModelPath, app };
+
 if (isDevelopment) {
-  webpackServeBundle(app);
+  webpackServeBundle(options);
```

Comment:
😱😱😱
A couple of simple, but very beneficial things for dev 💅 
Mainly using:
- [webpack-dev-middleware](https://webpack.github.io/docs/webpack-dev-middleware.html): compile assets in-memory and serve them
- [webpack-hot-middleware](https://github.com/glenjamin/webpack-hot-middleware): hot module reloading